staging/src/k8s.io/api/admission/v1alpha1/types.go

```diff
@@ -20,64 +20,84 @@ import (
 	authenticationv1 "k8s.io/api/authentication/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 )
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// AdmissionReview describes an admission request.
+// AdmissionReview describes an admission review request/response.
 type AdmissionReview struct {
 	metav1.TypeMeta `json:",inline"`
-	// Spec describes the attributes for the admission request.
-	// Since this admission controller is non-mutating the webhook should avoid setting this in its response to avoid the
-	// cost of deserializing it.
+	// Request describes the attributes for the admission request.
+	Request AdmissionRequest `json:"request,omitempty" protobuf:"bytes,1,opt,name=request"`
+	// Response describes the attributes for the admission response.
 	// +optional
-	Spec AdmissionReviewSpec `json:"spec,omitempty" protobuf:"bytes,1,opt,name=spec"`
-	// Status is filled in by the webhook and indicates whether the admission request should be permitted.
-	// +optional
-	Status AdmissionReviewStatus `json:"status,omitempty" protobuf:"bytes,2,opt,name=status"`
+	Response AdmissionResponse `json:"response,omitempty" protobuf:"bytes,2,opt,name=response"`
```

Comment:
The field name and protobuf tag changes throughout mean any existing webhook impl or protobuf client cannot correctly decode the context. 

I *think* that's acceptable given the API and feature was alpha, but worth a check against the deprecation policy,

Reply:
Yeah I think it should be OK.